apps/frontend/src/modules/education/keyboards/components/KeyboardCard.css

```
@@ -0,0 +1,84 @@
+    boxshadow: 0 0 0 0 rgba(59, 130, 246, 0.7);
+  }
+  50% {
+    border: 3px solid #2563eb;
+    box-shadow: 0 0 20px 5px rgba(59, 130, 246, 0.6);
+  }
+}
+
+@keyframes renderFlashUpdate {
+  0% {
+    border: 3px solid #10b981;
+    boxshadow: 0 0 0 0 rgba(16, 185, 129, 0.7);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)